Dockerfile-debian.template

```diff
@@ -20,6 +20,7 @@ ENV PYTHON_VERSION %%PLACEHOLDER%%
 
 RUN set -ex \
 	\
+	&& apt-get update && apt-get install -y --no-install-recommends libbluetooth-dev \
```

Comment:
Please note that I did not make anything regarding the  `savedAptMark` and `apt-mark showmanual`. 

I have very little clue about its meaning, but it seemed to be in the slim image.

Reply:
Sorry for not being more clear -- this one was fine before (with `libbluetooth-dev` up above `libssl-dev`'s comment line above), ala:

```diff
diff --git a/Dockerfile-debian.template b/Dockerfile-debian.template
index 4d56b25..8a14f49 100644
--- a/Dockerfile-debian.template
+++ b/Dockerfile-debian.template
@@ -9,6 +9,7 @@ ENV LANG C.UTF-8
 
 # extra dependencies (over what buildpack-deps already includes)
 RUN apt-get update && apt-get install -y --no-install-recommends \
+		libbluetooth-dev \
 # Python 3.4 on Stretch+ needs to use an older version of "libssl-dev" (these lines both get removed for every other combination)
 		libssl-dev \
 		tk-dev \
```

Happy to take over and make this change directly to your branch if you'd rather. :+1:

Thanks for updating this! :metal:

Reply:
Oh, my fault. Now that I re-read the comment it was clear that is was only referring to the slim image, not all Deb. 😅

I can also change it, since it's 5 min, but feel free to push to branch if you want :)
I'll stop force pushing.

Reply:
Done.

I force pushed one more time to keep the update.sh as the last commit.
I don't know what is the merge strategy, but maybe squash and merge would be beneficial here.

Also, does that libssl sill make sense? 
3.4 is not being rendered, so libssl never makes it to any of the images.

Reply:
Oh, nice catch!  Maybe we should clean that out separately from this though. :+1: